azure-storage-cpp-lite/src/blob/blob_client_wrapper.cpp

```diff
@@ -471,7 +478,7 @@ namespace microsoft_azure {
             std::condition_variable cv;
             std::mutex cv_mutex;
 
-            for(long long offset = 0, idx = 0; offset < fileSize; offset += UPLOAD_CHUNK_SIZE, ++idx)
+            for(long long offset = 0, idx = 0; offset < fileSize; offset += block_size, ++idx)
```

Comment:
Just checking - you tested locally?  We need to make some 'scale' tests - ones that we only run at ship time, but that test stuff like this.  For now, though, as long as we test locally with a >1TB file should be ok.

Reply:
Yes - 250GB and 1TB file tests succeeded. 1TB takes a loooong time due to us writing to the disk first, and then uploading to Blob.